cf/commands/serviceaccess/service_access.go

```diff
@@ -105,7 +105,7 @@ func (cmd *ServiceAccess) Run(c *cli.Context) {
 
 	brokers, err := cmd.actor.FilterBrokers(brokerName, serviceName, orgName)
 	if err != nil {
-		cmd.ui.Failed(T("Failed fetching service brokers.\n{{.Error}}", map[string]interface{}{"Error": err}))
+		cmd.ui.Failed(T("{{.Error}}", map[string]interface{}{"Error": err}))
```

Comment:
Since you are passing the error string as is, I fail to see why you'd i18n the string. So instead, until you have additional strings for format the error, the line above should be:

```
cmd.ui.Failed(err.Error())
```

I am assuming that this change is the correct thing to do. A different discussion.